relayer/rly/cosmos_relayer.go

```diff
@@ -23,6 +23,12 @@ type CosmosRelayer struct {
 
 func NewCosmosRelayer(log *zap.Logger, testName, home string, pool *dockertest.Pool, networkID string, options ...relayer.RelayerOption) *CosmosRelayer {
 	c := commander{log: log}
+	for _, opt := range options {
+		switch typedOpt := opt.(type) {
```

Comment:
`switch o := opt.(type)` would probably be a more typical name given the short scope of the variable here.